README.md

```diff
@@ -51,3 +51,14 @@ mkdir -p ~/Library/LaunchAgents
 ln -sfv /usr/local/opt/postgresql/*.plist ~/Library/LaunchAgents
 launchctl load -w ~/Library/LaunchAgents/homebrew.mxcl.postgresql.plist
 ```
+
+Setup a Twilio account
+
+- If you haven't already, sign up for an account at [twilio.com](https://www.twilio.com)
+- Create a configuration file for your application:
+`cp .env.example .env`
+- Replace TWILIO_ACCOUNT_SID, TWILIO_AUTH_TOKEN, and TWILIO_CALLER_ID with the appropriate values from your [account dashboard](https://www.twilio.com/user/account)
```

Comment:
Maybe specify buy a PN and / or verify a caller ID as step 2 and 3 (before filling out the values in the .env file?

Reply:
Good idea. Done!